package.json

```diff
@@ -5,21 +5,21 @@
     "build": "./node_modules/.bin/webpack"
   },
   "dependencies": {
-    "@babel/cli": "^7.6.2",
-    "@babel/core": "^7.6.2",
-    "@babel/preset-env": "^7.6.2",
+    "@babel/cli": "^7.7.4",
+    "@babel/core": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
     "babel-loader": "^8.0.6",
-    "core-js": "^3.2.1",
+    "core-js": "^3.4.7",
     "create-react-class": "^15.6.3",
     "prop-types": "^15.7.2",
-    "react": "^16.10.1",
-    "react-dom": "^16.10.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
     "react-dom-factories": "^1.0.2",
-    "react-redux": "^7.1.0",
-    "react-test-renderer": "^16.10.1",
+    "react-redux": "^7.1.3",
+    "react-test-renderer": "^16.12.0",
     "redux": "^4.0.4",
     "regenerator-runtime": "^0.13.3",
-    "webpack": "^4.41.0",
-    "webpack-cli": "^3.3.9"
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
```

Comment:
Are these dependency changes necessary to fix the issue? If not, I'd prefer to see these in a separate PR.

Reply:
they are not required.

Reply:
i mainly did it since i had to rebuild the bundles anyway.